inference-engine/tests/functional/plugin/shared/include/single_layer_tests/mat_mul.hpp

```diff
@@ -11,10 +11,20 @@
 
 #include "functional_test_utils/layer_test_utils.hpp"
 
+namespace LayerTestsDefinitions {
+namespace MatMulParams {
+enum class InputLayerType {
```

Comment:
We can move this enum to the `ngraph_helpers.hpp`. Similar enums are used for mat_mul and eltwise tests